product.php

```diff
@@ -0,0 +1,182 @@
+<?php include 'includes/header.php'; ?>
+
+
+<main>
+<!--Internal Product Navigation Links-->
+    <div class="row"> <!--one-->
+        <div class="twelve columns">
+            <nav>
+                <a href="">Home</a> >>
+                <a href="">Dogs</a> >>
+                <a href="">Food</a> >>
+                <a href="">Desserts</a>
+            </nav>
+        </div>
+    </div>
+
+    <!--Product Name-->
+    <div class="row"> <!--two-->
+        <div class="six columns">
+            <h1>Product Name</h1>
+        </div>
+
+        <!--BLANK SIX CLOUMNS-->
+
+    </div>
+
+    <!--Product Image and purchasing information-->
+    <div class="row"> <!--three-->
+        <div class="six columns">
+            <figure>
+                <img src="https://images.pexels.com/photos/913136/pexels-photo-913136.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=650&w=940" style="width:300px; height:400px" title="Main Image" alt"Sweet-tart cupcake"/>
+                <!--<figcaption></figcaption>-->
+            </figure>
+        </div>
+        <div class="six columns">
+            <img src="https://cdn1.iconfinder.com/data/icons/ecommerce-v1-0-outline-1/96/Review_Star-128.png" style="width:50px; height:50px" title="Ratings" alt="Three-star rating"/>
+            <a href="#reviews">Reviews</a>
+
+            <br>
+
+            <p>Price: $ price variable</p>
+
+            <form action="" method="POST">
+                <label>Quantity
```

Comment:
Make sure to add the for attribute to link the label to the quantity `<select>`

Reply:
It was added

Reply:
I'm still not seeing it? It should be `<label for="quantity">`